phpunit/class-rest-widget-types-controller-test.php

```diff
@@ -264,6 +265,77 @@ public function test_get_widget_form() {
 		$this->assertArrayHasKey( 'form', $data );
 	}
 
+	public function test_encode_form_data_with_no_input() {
```

Comment:
Sorry if I'm missing it, but do we have a test where the hash doesn't match?

Reply:
Yes, `test_create_item_malformed_instance`.